Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -2,6 +2,7 @@ import KsApi
 import Prelude
 import ReactiveExtensions
 import ReactiveSwift
+import Stripe
```

Comment:
We should avoid coupling 3rd party SDKs with our `Library` framework to limit dependencies. Ideally you would pass the info needed from the VC to the VM so you don't have to include this import. One way to do that would be to extend the `CardDetails` typealias to take a `cardBrand: GraphUserCreditCard.CreditCardType`, and then pass the card type from the `STPPaymentCardTextFieldDelegate` function:

```
// From AddNewCardViewController
  internal func paymentCardTextFieldDidChange(_ textField: STPPaymentCardTextField) {
    self.viewModel.inputs.paymentInfo(isValid: textField.isValid)

    guard let cardnumber = textField.cardNumber else {
      return
    }

    let stpCardBrand = STPCardValidator.brand(forNumber: cardnumber)

    let cardDetails = (cardnumber,
                       textField.expirationMonth,
                       textField.expirationYear,
                       textField.cvc,
                       stpCardBrand.creditCardType)

    self.viewModel.inputs.creditCardChanged(cardDetails: cardDetails)
  }
```

You'll want to somehow map `STPCardBrand` -> `GraphUserCreditCard.CreditCardType`, but you could probably do that in an extension.